clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListColumns.java

```diff
@@ -67,8 +67,12 @@ public void handle() {
       String name = columns[i].name();
       String dataType = columns[i].dataType().simpleString();
       String comment = columns[i].comment();
-      String nullable = columns[i].nullable() ? "null" : "";
-      String autoIncrement = columns[i].autoIncrement() ? "auto" : "";
+      String nullable = columns[i].nullable() ? "true" : "false";
+      String autoIncrement = columns[i].autoIncrement() ? "true" : "false";
+
+      if (i == 0) {
+        all.append("name,datatype,comment,nullable,auto" + System.lineSeparator());
+      }
```

Comment:
1) I think you can move this line above, out of "for loop", so you don't have to check if `i == 0`;
2) It is better to change "auto" to "auto_increment" for better understanding.

Reply:
1. Not really, as you don't want to output the header if there is no data.
2. Sure I do that.

Reply:
Justin, the column header name is still "auto", not "auto_increment", can you confirm it?

Reply:
"I'll do that" means I do that in the future, not that it has been done. It has been done now.